utilities/release-osci.py

```diff
@@ -0,0 +1,277 @@
+#!python3
+
+import argparse
+import shutil
+import os
+import subprocess
+import yaml
+
+
+g_config_root = "ci-operator/config/stackrox/collector"
+g_steps_root = "ci-operator/step-registry/stackrox/collector"
+
+
+def config(name):
+    return os.path.join(g_config_root, name)
+
+
+def step_file(*parts):
+    return os.path.join(g_steps_root, *parts)
+
+
+class OpenshiftRelease:
+    def __init__(self, root, version, dry_run=False):
+        self.root = root
+        self.dry_run = dry_run
+        self.version = version
+        self.version_str = f"release-{version}"
+
+    def from_root(self, *paths):
+        """
+        Helper for getting a path from the root of the repository
+
+        e.g. from_root('ci-operator/config') -> /path/to/repo/ci-operator/config
+        """
+        return os.path.join(self.root, *paths)
+
+    def step_from_root(self, *paths):
+        """
+        Helper for getting a file relative to stackrox/collector's step
+        registry.
+
+        e.g. step_from_root('release-3.11/integration-tests')
+             -> /path/to/repo/ci-operator/step-registry/stackrox/collector/release-3.11/integration-tests
+        """
+        return self.from_root(step_file(*paths))
+
+    def config_from_root(self, name):
+        """
+        Helper for getting a config file relative to stackrox/collector's config
+        directory.
+
+        e.g. config_from_root('stackrox-collector-master.yaml')
+             -> /path/to/repo/ci-operator/config/stackrox/collector/stackrox-collector-master.yaml
+        """
+        return self.from_root(config(name))
```

Comment:
These can probably be written as:
```py
return self.from_root(g_config_root, name)
```
And we can directly remove `config` since it is not used elsewhere AFAICT.

Reply:
Done